src/nano_contracts/deserializer.ts

```diff
@@ -263,78 +299,54 @@ class Deserializer {
    * @memberof Deserializer
    * @inner
    */
-  toSigned(signedData: Buffer, type: string): DeserializeResult<string> {
-    const [containerType, internalType] = getContainerInternalType(type);
-    if (containerType !== 'SignedData') {
-      throw new Error('Type is not SignedData');
-    }
-    if (!internalType) {
-      throw new Error('Unable to extract type');
-    }
-    // Should we check that the valueType is valid?
-
-    // SignData[T] is serialized as Serialize(T)+Serialize(sign(T)) where sign() returns a byte str
-    // Which means we can parse the T argument, then read the bytes after.
+  toSignedData(signedData: Buffer, type: string): BufferROExtract<NanoContractSignedData> {
```

Comment:
The `type` argument changed from being the whole `SignedData[innerType]` to just the inner type

Reply:
Done